### PR TITLE
Added the parameters for the Nearest Centroid Classifier

### PR DESCRIPTION
**Pull request by Tanuj2552**
By this commit will fix the issue of #47 
Added the parameters for the Nearest Centroid Classification Model and tested it on the Pima Indians Diabetes Dataset (from 3rd data set present in the website https://machinelearningmastery.com/standard-machine-learning-datasets/ )

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
@Tanuj2552 
the model doesnt have l1 and l2 has parameter for metric
do refer official api parameters
https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.NearestCentroid.html?highlight=nearestcentroid#sklearn.neighbors.NearestCentroid

**Comment by Tanuj2552** [comment]
Well, actually it does have those parameters. Those are not directly given in that nearest centroid API, but if we go through it, it says that it has the same metrics as for metrics.pairwise.pairwise_distances. Then I checked that [API](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances.html) and it clearly states that these parameters are available, and I have verified it by testing it with a classification dataset. 
Please check once again and let me know.

**Comment by Thilakraj1998** [comment]
@Tanuj2552 then do add manhattan and euclidean metric too

**Comment by Tanuj2552** [comment]
yeah sure.
so for that, should I again make a new pull request?
